### PR TITLE
use ogr

### PR DESCRIPTION
**Pull request by TomasTomecek**
* port upsint to ogr
* use the same configuration format as packit and ogr
* config file can now be json and yaml (even falls back to packit.yaml)
* create-pr: do `git push -q -u`

this should mostly work now

needs https://github.com/packit-service/ogr/pull/304 for list-tags

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 10
           

Complexity increasing per file
==============================
- upsint/exceptions.py  1
         

Complexity decreasing per file
==============================
+ upsint/core.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/user-cont/tool/pullRequest?prid=4752755&bid=15853028)

**Comment by lachmanfrantisek** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           

Complexity increasing per file
==============================
- upsint/conf.py  1
- upsint/exceptions.py  1
         

Complexity decreasing per file
==============================
+ upsint/core.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/user-cont/tool/pullRequest?prid=4752755&bid=15853028)

File: upsint/cli.py [hunk 1]
```diff
@@ -31,17 +34,35 @@ def upsint():
...
+        git_project = app.get_git_project(f"https://github.com/{repo}")
+    username = git_project.service.user.get_username()
+    forked_repo = git_project.fork_create()
+    # FIXME: haxxxxx
+    forked_repo.repo = target_repo_name
+    forked_repo.namespace = username
+
+    forked_repo_ssh_url = forked_repo.get_git_urls()["ssh"]
+
+    clone_repo_and_cd_inside(target_repo_name, forked_repo_ssh_url, forked_repo.namespace)
```
Comment (TomasTomecek) [inline upsint/cli.py]: TODO: clone into upstream namespace name, not target name

**Comment by lachmanfrantisek** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 24
           

Complexity increasing per file
==============================
- upsint/exceptions.py  1
- upsint/conf.py  1
         

Complexity decreasing per file
==============================
+ upsint/core.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/user-cont/tool/pullRequest?prid=4752755&bid=15853028)

**Pull request merged**